src/addresses/address-details/AddressDetails.container.tsx

```diff
@@ -38,6 +39,7 @@ const AddressDetailsPage: FC = () => {
   const [brokerDetails, setBrokerDetails] = useState<K8sResourceKind>({});
   const [_loading, setLoading] = useState<boolean>(true);
   const [isFirstMount, setIsFirstMount] = useState(true);
+  const [error, setError] = useState<string | null>(null);
```

Comment:
Use an empty string instead of null to initialize a state 

`  const [error, setError] = useState<string>('');`